pages/common/crictl.md

```diff
@@ -0,0 +1,36 @@
+# crictl
+
+> Command-line for CRI-compatible container runtimes.
+> More information: <https://github.com/kubernetes-sigs/cri-tools/blob/master/docs/crictl.md>.
+
+- List all kubernetes pods(Ready and NotReady):
+
+`crictl pods`
+
+- List all containers (Running and Exited):
```

Comment:
Should be in upper case?
```suggestion
- List all containers (running and exited):
```

Reply:
> Should be in upper case?

When I executed this command, the result list show me the state is `Running` or  `Exited `, so I thought it would be in upper case. Do you think it's okay?